tests/safeds/data/image/containers/test_image.py

```diff
@@ -57,9 +57,9 @@ def test_should_return_correct_format(self, image: Image, format_: ImageFormat)
 class TestToJpegFile:
     @pytest.mark.parametrize(
         "path",
-        ["image/white_square.jpg"],
+        ["image/white_square.jpg", Path("image/white_square.jpg")],
```

Comment:
The extra test case is not needed since `path` is only used for the `from_jpeg_file` method, which is tested separately. Similarly to the tests for `to_csv_file` and `to_json_file`, you'll need to copy the entire test and pass a path to `to_jpeg_file`.

Reply:
added separate tests.